plugins/src/test/config/conf/contextTestParams.properties

```diff
@@ -1,45 +1,45 @@
-######## Datasources -- start
-
-# IMPORTANT #
-# any other datasource has to have parameters likes the follows:
-# jdbc.<DS_TEST_CODE>.beanName
-# jdbc.<DS_TEST_CODE>.driverClassName
-# jdbc.<DS_TEST_CODE>.url
-# jdbc.<DS_TEST_CODE>.username
-# jdbc.<DS_TEST_CODE>.password
-# DS_TEST_CODE is the unique code of the datasource
-
-jdbc.portDb.beanName=portDataSource
-jdbc.portDb.driverClassName=${test.database.driver}
-jdbc.portDb.url=${test.database.port.url}
-jdbc.portDb.username=${test.database.username}
-jdbc.portDb.password=${test.database.password}
-
-jdbc.servDb.beanName=servDataSource
-jdbc.servDb.driverClassName=${test.database.driver}
-jdbc.servDb.url=${test.database.serv.url}
-jdbc.servDb.username=${test.database.username}
-jdbc.servDb.password=${test.database.password}
-
-######## Datasources -- end
-
-logActiveFileOutput=false
-logName=${project.artifactId}
-logFilePrefix=${project.build.directory}/test/log_Entando
-logLevel=ERROR
-logFileSize=512000
-logFilesCount=10
-
-configVersion=test
-
-applicationBaseURL=/Entando/
-resourceRootURL=/Entando/resources/
-protectedResourceRootURL=/Entando/protected/
-
-resourceDiskRootFolder=${project.build.directory}/test/resources/
-protectedResourceDiskRootFolder=${project.build.directory}/test/protected/
-
-indexDiskRootFolder=${project.build.directory}/test/indexdir
-
-portDataSourceClassName=${test.database.driver}
-servDataSourceClassName=${test.database.driver}
+######## Datasources -- start
+
+# IMPORTANT #
+# any other datasource has to have parameters likes the follows:
+# jdbc.<DS_TEST_CODE>.beanName
+# jdbc.<DS_TEST_CODE>.driverClassName
+# jdbc.<DS_TEST_CODE>.url
+# jdbc.<DS_TEST_CODE>.username
+# jdbc.<DS_TEST_CODE>.password
+# DS_TEST_CODE is the unique code of the datasource
+
+jdbc.portDb.beanName=portDataSource
+jdbc.portDb.driverClassName=${test.database.driver}
+jdbc.portDb.url=${test.database.port.url}
+jdbc.portDb.username=${test.database.username}
+jdbc.portDb.password=${test.database.password}
+
+jdbc.servDb.beanName=servDataSource
+jdbc.servDb.driverClassName=${test.database.driver}
+jdbc.servDb.url=${test.database.serv.url}
+jdbc.servDb.username=${test.database.username}
+jdbc.servDb.password=${test.database.password}
+
+######## Datasources -- end
+
+logActiveFileOutput=false
+logName=${project.artifactId}
+logFilePrefix=${project.build.directory}/test/log_Entando
+logLevel=ERROR
+logFileSize=512000
+logFilesCount=10
+
+configVersion=test
+
+applicationBaseURL=/Entando/
+resourceRootURL=/Entando/resources/
+protectedResourceRootURL=/Entando/protected/
+
+resourceDiskRootFolder=target/test/resources/
```

Comment:
Is this change intentional?

Reply:
Yes, I need to allow the test environment to read the cms resources (added for test) from the filesystem to allow tika (new version) to index the content of the file